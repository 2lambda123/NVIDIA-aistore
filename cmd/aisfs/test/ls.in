touch $DIR/a.txt
mkdir $DIR/b
touch $DIR/b/c.txt
touch $DIR/b/d.txt

ls -R $DIR

mkdir -p $DIR/x/{a,b}/{e,f,g}/{h,i,j}

ls $DIR/x/b/g
ls $DIR/x/b/g/h
ls $DIR/x/b/g/h/i // FAIL "no such file or directory"

touch $DIR/x/b/g/h/i.txt
ls $DIR/x/b/g/h
rm -r $DIR/x
ls $DIR/x // FAIL "no such file or directory"
