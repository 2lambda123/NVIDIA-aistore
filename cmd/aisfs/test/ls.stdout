$DIR:
  a.txt
  b

$DIR/b:
  c.txt
  d.txt
h
i
j
i.txt
